front/components/sparkle/AppLayoutTitle.tsx

```diff
@@ -36,16 +45,37 @@ export function AppLayoutTitle({
             />
           </div>
         )}
-        {action && (
```

Comment:
Why remove action entirely without removing the component argument?

Reply:
No good reason, I'll re-set it as it might be useful elsewhere! 🙏🏻